scripts/install.sh

```diff
@@ -39,9 +39,12 @@ KEEP_CORE_SOL_ARTIFACTS_PATH=$(realpath $KEEP_CORE_SOL_PATH/build/contracts)
 printf "${LOG_START}Initializing contracts...${LOG_END}"
 truffle exec scripts/delegate-tokens.js --network local
 
-printf "${LOG_START}Updating keep-core client config...${LOG_END}"
-KEEP_CORE_CONFIG_FILE_PATH=$KEEP_CORE_CONFIG_FILE_PATH \
-    truffle exec scripts/lcl-client-config.js --network local
+config_files=($KEEP_CORE_CONFIG_DIR_PATH/*.toml)
+printf "${LOG_START}Updating keep-core client configs...${LOG_END}"
+for f in "${config_files[@]}"; do
+    KEEP_CORE_CONFIG_FILE_PATH="/${f##/}" \
```

Comment:
What's the advantage of this approach over the one in `keep-ecdsa`:
```
for CONFIG_FILE in $KEEP_ECDSA_CONFIG_DIR_PATH/*.toml
do
  KEEP_ECDSA_CONFIG_FILE_PATH=$CONFIG_FILE \
```
?
🤔

Reply:
No particular reason. I didn't notice we have similar loop at `keep-ecdsa/initialize.sh` and just implemented my own `for-each`.

Refactored to the one we currently have in `keep-ecdsa`.